test/integration/variable_test.rb

```diff
@@ -79,4 +79,10 @@ def test_hash_with_default_proc
   def test_multiline_variable
     assert_equal 'worked', Template.parse("{{\ntest\n}}").render!('test' => 'worked')
   end
+
+  def test_global_options_on_variable
+    template = Template.parse(%({{test}}), global_filters: ['truncate: 5'])
```

Comment:
I don't see why liquid syntax should be used to define global filters.  I think taking a ruby proc would make more sense, and I think this could be used to implement taint checking.